neurodsp/spectral/variance.py

```diff
@@ -148,7 +148,7 @@ def compute_scv_rs(sig, fs, window='hann', nperseg=None, noverlap=0,
 
         # Repeated sub-sampling of spectrogram randomly, with replacement between draws
         for draw in range(ndraws):
-            idx = np.random.choice(spg.shape[1], size=nslices, replace=False)
+            idx = np.random.choice(spg.shape[1], size=nslices, replace=True)
```

Comment:
Hmm, good catch, this is totally a disconnect between comment & actual code. 

I'm tagging in @rdgao to confirm that the doc is correct, and this sampling is intended to be replacement, and so code should be updated here.

Reply:
I realized shortly after this commit that this doesn't affect anything. This is drawing one sample from the list of indices, so whether or not use draw with replacement the distribution is unaffected. Apologies for the confusion.